docs/userGuide/boilerplate.md

```diff
@@ -0,0 +1,24 @@
+The `_markbind/boilerplates` folder in the site's root directory can be used to store repetitive files. You might have a file `structure.md` (that contain text such as `<include src="text.md">`) that needs to be used in many places. You want the `text.md` to include to depend on the folder `structure.md` is supposed to be in (in this case, `book/architecture/full.md`). Instead of duplicating the file in different locations, you will only need a copy in the boilerplate directory - the final outcome is as if the file was present in the different locations.
+
```

Comment:
Is there a link to this page in the markbind user guide?

Reply:
https://github.com/MarkBind/markbind/pull/191/files#diff-1b36e91bdf5dc0e4a02bd693e881b8a5R32

Reply:
I feel that **Boilerplates** deserves its own point; it was released as a Feature in [v1.5.0](https://github.com/MarkBind/markbind-cli/releases/tag/v1.5.0).